.github/workflows/iroha2-dev.yml

```diff
@@ -19,10 +17,22 @@ jobs:
       image: 7272721/i2-ci:nightly
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - uses: tj-actions/changed-files@v29.0.2
+        id: changed-files
       - uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push iroha2-base image
+        if: contains(steps.changed-files.outputs.modified_files, 'Dockerfile.base')
+        uses: docker/build-push-action@v3
+        with:
+          push: true
+          tags: hyperledger/iroha2-base:dev
+          labels: commit=${{ github.sha }}
+          file: Dockerfile.base
```

Comment:
```suggestion
      - uses: dorny/paths-filter@v2.10.2
        id: filter
        with:
          filters: |
            base-image:
              - 'Dockerfile.base'
      - uses: docker/login-action@v2
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
      - name: Build and push iroha2-base image
        if: steps.filter.outputs.base-image == 'true'
        uses: docker/build-push-action@v3
        with:
          push: true
          tags: hyperledger/iroha2-base:dev
          labels: commit=${{ github.sha }}
          file: Dockerfile.base
```

But I have another one error with this action that we don't have with `load-rs` repo:
```
Run dorny/paths-filter@v2.10.2
/usr/bin/docker exec  70593ac1b69ec16d10ebedea742c0cf84665d9971872fa7e56001cb05bbb612d sh -c "cat /etc/*release | grep ^ID"
Get current git ref
  /usr/bin/git branch --show-current
  fatal: detected dubious ownership in repository at '/__w/soramitsu-iroha/soramitsu-iroha'
  To add an exception for this directory, call:
  
  	git config --global --add safe.directory /__w/soramitsu-iroha/soramitsu-iroha
Error: The process '/usr/bin/git' failed with exit code [12](https://github.com/BAStos525/soramitsu-iroha/runs/8169021883?check_suite_focus=true#step:4:13)8
```
Could you try @s8sato

Reply:
https://github.com/hyperledger/iroha/pull/2680#discussion_r964408423